picard/coverart/processing/__init__.py

```diff
@@ -61,16 +63,18 @@ def run_image_processors(data, coverartimage):
         both_queue, tags_queue, file_queue = get_cover_art_processors()
         for processor in both_queue:
             processor.run(image, ProcessingTarget.BOTH)
-        tags_image = image.copy()
-        for processor in tags_queue:
-            processor.run(tags_image, ProcessingTarget.TAGS)
-        tags_data = tags_image.get_result(default_format=True)
+        if config.setting['save_images_to_tags']:
```

Comment:
If "Embed cover images into tags" is disabled, we still have the image shown in New Cover (unprocessed), that's a bit misleading I think.

![Capture d’écran du 2024-06-15 12-18-53](https://github.com/metabrainz/picard/assets/151042/912b3358-d345-4939-b56a-cadc61125bd2)

![Capture d’écran du 2024-06-15 12-19-31](https://github.com/metabrainz/picard/assets/151042/a1662f1a-e843-4456-8da4-1350da8db698)